external/loaders/loaders/mappers/_hybrid.py

```diff
@@ -38,6 +38,7 @@ def __getitem__(self, key: str) -> xr.Dataset:
 T_Mapper = FineResolutionResidual
 
 
+@no_type_check
```

Comment:
Any particular reason to skip this one? I don't think it's pending deletion.

Reply:
It's substantially refactored in my next PR, where I've removed all the remaining no_type_checks added in this one.